src/main/java/io/aiven/kafka/connect/common/output/avro/AvroConfig.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2023 Aiven Oy
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.aiven.kafka.connect.common.output.avro;
+
+import java.util.Map;
+import java.util.Objects;
+
+import org.apache.kafka.common.config.AbstractConfig;
+import org.apache.kafka.common.config.ConfigDef;
+import org.apache.kafka.common.config.ConfigException;
+
+import org.apache.avro.AvroRuntimeException;
+import org.apache.avro.file.CodecFactory;
+
+final class AvroConfig extends AbstractConfig {
+
+    private static final String GROUP_AVRO = "Avro";
+    private static final String AVRO_CODEC_CONFIG = "avro.codec";
+    private static final String DEFAULT_AVRO_CODEC = "null";
```

Comment:
Should we refer to `org.apache.avro.file.DataFileConstants.NULL_CODEC` here?

Reply:
Done. Changed also the `AVRO_CODEC_CONFIG = DataFileConstants.CODEC`.